base/qaqc/R/cull_database_entries.R

```diff
@@ -17,63 +17,50 @@
 ##' For more information on how to use this function see the "Pre-release-database-cleanup" script in the 'vignettes' folder
 ##' or look at the README
 ##'
-cull_database_entries<-function(table = NULL, outdir, file_name = NULL, con, machine_id = NULL, table_name = NULL){
-  
-  
-  if(is.null(table)){
-    
-    if(is.null(file_name)){
-      
+cull_database_entries <- function(table = NULL, outdir, file_name = NULL, con, machine_id = NULL, table_name = NULL) {
+  if (is.null(table)) {
```

Comment:
Thank you for including the stylr run, and especially for doing it as its own commit at the end of the PR! (That makes it easy for us to review the substantive changes separately from any widespread-but-minor formatting fixes)👍